organization/hsf-letters.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Letters from the HSF"
+author: "Graeme Stewart"
+layout: default
+---
+
+# {{page.title}}
+
+The HSF's mission is to encourage cooperation and common projects
+between all parts of the high-energy physics community concerned
+with software. We span regions and experiments and we strive to 
+to make our community's software development more effective.
+To that end, the HSF will happily cooperate with any project that
+shares similar goals. 
+
+### Letters of Cooperation
+
+We will be happy to provide a *letter of cooperation*
+that can be used for any funding application. To ask us
```

Comment:
```suggestion
that can be used for any software project and/or funding application. To ask us
```

Reply:
Just trying to make the case more broad, so not just funding applications. What do you think?

Reply:
Yes, we should avoid being too specific - thanks!

Reply:
I just used a different wording to avoid the ugly "and/or"

Reply:
Fair enough :-).